site/content/blog/simplifying-package-authoring.md

```diff
@@ -0,0 +1,301 @@
+---
+title: "Simplifying Package Authoring"
```

Comment:
```suggestion
title: "Simplifying Package Authoring with kctrl"
```

Reply:
Received two titles as suggestion:
`Simplifying Package Authoring with kctrl` or `Kubernetes Package Authoring with kctrl`. As more people have added +1 for `Kubernetes Package Authoring with kctrl`, going with `Kubernetes Package Authoring with kctrl`